vulkan_generator/generator.py

```diff
@@ -14,12 +14,16 @@
 
 """This is the top level point for Vulkan Code Generator"""
 
+import os
+
 from pathlib import Path
 import pprint
 
 from vulkan_generator.vulkan_parser import parser as vulkan_parser
 from vulkan_generator.vulkan_parser import types
 
+from handle_remapper import generator as handle_remapper
```

Comment:
I think the name after as should not be same with the package name. It may confuse both tools and the reader.

Reply:
This will be solved by b/236110214 when I separate the different code generators into separate python binaries. Leaving as is now because otherwise the vulkan_generator/generator.py file has no reason to exist outside of your testing and experiments.